### ISSUE TITLE
Error when pushing tags

### ISSUE THREAD

**Issue by ethanwhite** [weecology/PortalData#279]
`archive.R` is currently erroring due to a failure by `git2r` to push the tags to the repo. This has the cascading consequence of preventing a new release and archiving to Zenodo. Error:

```
Error in git2r::push(repo, name = "deploy", refspec = paste("refs/tags/",  : 
  Error in 'git2r_push': object not found - no match for id (3a171428923f3eef21558ff18a2470f14d003392)
``` 

Our current usage follows https://github.com/ropensci/git2r/issues/265, so it's not clear if `git2r` changed or if something else in the transition to `archive.R` broke something.

**Comment by ethanwhite** [comment]
The good/bad news is that this is working properly in https://github.com/weecology/livedat using identical code. Possible differences that could result in different behavior:

1. Different cached versions of git2r for different Travis builds
2. Libraries are loaded in livedat rather than accessed with `::` as they are in PortalData
3. Some difference in the setup connecting Travis and GitHub. GitHub Apps didn't exist when we started so the process of connecting was different.

**Comment by ethanwhite** [comment]
This warning is currently showing up in `livedat` after flushing the package cache:

```
Unable to find the libgit2 library on this system. Building 'git2r'
   using the bundled source of the libgit2 library.
```

But `livedat` runs successfully even after flushing the cache.

**Comment by ethanwhite** [comment]
And... I can't migrate this repo to the new Travis system because it's on the old Travis system and the ability to migrate has been pushed back by ~9 months already...

**Comment by ethanwhite** [comment]
Fixed in eeba493569824dce079600f0f06fc33651636c01 by running the git push from the command line using system. With that the build, at least for `master` is now fixed and producing archived data on Zenodo again.
